### PR TITLE
Add CLI reference documentation to new site

### PR DESCRIPTION
**Pull request by nicholastmosher**
This supersedes #84 now that the new site has been merged.

### CODE REVIEW

File: assets/styles/_highlight.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* Background */ .chroma { font-size: .9em; padding-left:10px !important; color: #f8f8f2; background-color: $code-background !important; line-height:1.65em;}
+/* Background */ .chroma { font-size: .9em; padding-left:10px !important; color: #f8f8f2; box-shadow: 5px 5px 10px lightgrey; border-radius: 5px; background-color: $code-background !important; line-height:1.65em;}
```
Comment (sehz) [inline assets/styles/_highlight.scss:1]: is this intentional?

Comment (ajhunyady) [reply assets/styles/_highlight.scss:1]: I'll share my opinion once I see the look.

**Comment by ajhunyady** [comment]
@nicholastmosher, let's merge this if ready.

**Comment by nicholastmosher** [comment]
I think there are a few more points you made in the other PR that I haven't gotten to yet, but if you want we can merge this now if it's good enough and we can iterate on the other bits

File: assets/styles/_highlight.scss (hunk 1 above)
Comment (sehz) [inline assets/styles/_highlight.scss:1]: separate out CSS changes as different PR

**Review by sehz — CHANGES_REQUESTED** [review]

File: content/cli/commands/cluster.md [hunk 2]
```diff
@@ -1,5 +1,274 @@
...
-menu: Runner
-weight: 10
+title: Fluvio Cluster
+menu: Cluster
+weight: 50
 ---
+
+The `fluvio cluster` family of commands is used to install and manage your own
+Fluvio cluster. The two primary forms of self-hosted cluster are Kubernetes
+(via Minikube) and Local clusters, both of which can be automatically set up
```
[the remark this answers is not in the document]
Comment (nicholastmosher) [reply content/cli/commands/cluster.md:9]: I mean, our current `fluvio cluster` commands do have minikube-specific ties. But if you want, I can change this to be something like "e.g. via Minikube" to imply it does not need to be minikube

File: content/cli/commands/cluster.md [hunk 3]
```diff
@@ -1,5 +1,274 @@
...
+### Example usage:
+
+To start a cluster on Minikube:
+
+%copy first-line%
+```bash
+$ fluvio cluster start
+"fluvio" has been added to your repositories
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "jaegertracing" chart repository
```
[the remark this answers is not in the document]
Comment (nicholastmosher) [reply content/cli/commands/cluster.md:143]: Hmm interesting. This happened on my local machine because I added the jaegertracing repository at one point, and I missed that fact when copying and pasting. This is not the typical behavior, just my mistake. Will fix

File: content/cli/commands/cluster.md [hunk 4]
```diff
@@ -1,5 +1,274 @@
...
-title: Fluvio Runner
-menu: Runner
-weight: 10
+title: Fluvio Cluster
+menu: Cluster
+weight: 50
 ---
+
+The `fluvio cluster` family of commands is used to install and manage your own
+Fluvio cluster. The two primary forms of self-hosted cluster are Kubernetes
```
[the remark this answers is not in the document]
Comment (nicholastmosher) [reply content/cli/commands/cluster.md:8]: What do you think about "self-managed"? I am trying to find a term that distinguishes it from Fluvio Cloud.

As in, "on one hand you can use Fluvio Cloud, or on the other hand you can use a _ cluster"

- self-hosted?
- self-managed?
- self-run?

What do you think?

[the remark this answers is not in the document]
Comment (sehz) [reply content/cli/commands/cluster.md:8]: With Fluvio Cloud, Infinyon Manages infrastructure for you (similar to AWS).   With `cluster` command, you can build and operate your own Fluvio Cluster

File: content/cli/commands/cluster.md [hunk 5]
```diff
@@ -1,5 +1,274 @@
...
+You may proceed with cluster startup
+next: run `fluvio cluster start`
+```
+
+## `fluvio cluster start`
+
+This command is used to start your own Fluvio cluster, with all the
+machinery needed to receive, process, and serve streaming messages.
+
+There are two main variations of this command. The default variation
```
[the remark this answers is not in the document]
Comment (nicholastmosher) [reply content/cli/commands/cluster.md:58]: Right... I think this paragraph explains that pretty well? Is there something specific you'd like me to change?

[the remark this answers is not in the document]
Comment (sehz) [reply content/cli/commands/cluster.md:57]: Hmm. Ok. this is probably good enough for now

**Review by sehz — APPROVED** [review]
